sklearn/metrics/_plot/roc_curve.py

```diff
@@ -53,14 +54,15 @@ class RocCurveDisplay:
     >>> display.plot()  # doctest: +SKIP
     >>> plt.show()      # doctest: +SKIP
     """
-
-    def __init__(self, fpr, tpr, roc_auc, estimator_name):
+    @_deprecate_positional_args
+    def __init__(self, *, fpr, tpr, roc_auc, estimator_name):
```

Comment:
* after tpr

Reply:
I'd agree that putting `*` after `precision` and `recall` would maybe be ok since the class is called `PrecisionRecall...`, but here there's no way to know which one's supposed to come first.

Reply:
Actually, not sure. Because the order it comes first in corresponds to roc_curve output, so it should be easy to use `RocCurveDisplay(*roc_curve_output, ...)`

Reply:
Yes, but not sure how many people would write it as you suggest, and it probably looks a bit cryptic to most people.